metals/src/main/scala/scala/meta/internal/metals/BuildServerConnection.scala

```diff
@@ -313,7 +328,7 @@ object BuildServerConnection {
         if (isCI)
           initializeResult.get(20, TimeUnit.SECONDS)
         else
-          initializeResult.get(5, TimeUnit.SECONDS)
+          initializeResult.get(205, TimeUnit.SECONDS)
```

Comment:
205 might be too much and should only be needed in case of sbt. Also, if it might take a longer time we could show the progress?

Reply:
True. I changed around the logic a bit and if we are trying to connect to sbt, I'll give it a minute. We can make that a bit larger if need be, but probably good to start. Then if it's not sbt, I got rid of the CI check and I just default to 30 seconds.